index.js

```diff
@@ -96,7 +100,11 @@ function messagesToJson(messages, options) {
         ruleId: message.ruleId || null,
         source: message.source || null,
         fatal: message.fatal,
-        stack: message.stack || null
+        stack: message.stack || null,
+        actual: message.actual || null,
```

Comment:
Hey! Have you thought about using `undefined` as a fallback? That way, they don’t show up in the JSON.
Feels like it makes a bit more sense to me for these non-standard fields. What do you think?